CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 2.32.2.20240516 Linux re-build
+
+*This release has the same Fluent Bit contents as 2.32.0, and is simply a linux-only re-build for recent patches in dependencies installed in the image. There are no windows images for this release.* 
```

Comment:
could you write "...contents as 2.32.2". That's really the same but it's more appropriate to use the most recent version.

Reply:
Modified this and the previous CVE release to use 2.32.2.